generator-core/src/main/java/br/com/generator/core/GeneratorExecutor.java

```diff
@@ -35,7 +35,7 @@ public File addDependecies(GeneratorOptions options) throws IOException {
         File fileGenerated = new File(options.getDestination());
         FileUtils.writeStringToFile(fileGenerated, contentReplaced);
 
-        System.out.println("Add dependencies in ".concat(options.getDestination()));
+        System.out.println("Add dependencies in ".concat(options.getName()));
```

Comment:
The two print statements seem to be doing similar things, consider extracting a method to avoid duplication.